micro/src/test/scala/example/TypedHelloSpec.scala

```diff
@@ -82,7 +82,7 @@ class TypedHelloSpec extends ScalatraFlatSpec {
 
   it should "return cookies in the ActionResult" in {
     get("/cookie") {
-      header("Set-Cookie") should equal("foo=bar;Path=/")
+      header("Set-Cookie") should equal("foo=bar; Path=/")
```

Comment:
Hahm, has Jetty changed the header value (probably again), right?

Reply:
Yes